testhelpers/docker_registry.go

```diff
@@ -182,6 +193,39 @@ func (r *DockerRegistry) EncodedLabeledAuth() string {
 	return base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"username":"%s","password":"%s"}`, r.username, r.password)))
 }
 
+// SetPrivilegesToImage sets the given image name with the provided ImagePrivileges if ImagePrivileges was enabled.
+// For example SetPrivilegesToImage("my-image", NewImagePrivileges(Readable, Writable)) will save "my-image" as a
+// readable and writable image into the registry.
+func (r *DockerRegistry) setPrivilegesToImage(imageName string, privilege ImagePrivileges) {
+	if r.isImagePrivilegesEnable() {
+		r.imagePrivileges[imageName] = privilege
+	}
+}
+
+// MakeReadOnly set the given image name to be readable when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeReadOnly(imageName string) {
+	r.setPrivilegesToImage(imageName, NewImagePrivileges(Readable))
```

Comment:
```suggestion
	r.setImagePrivileges(imageName, NewImagePrivileges(Readable))
```
(nit)

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/c59c94c0966e2826f3e1ce6b48983be810952fee)